corehq/pillows/mappings/app_mapping.py

```diff
@@ -4,7 +4,7 @@
 from corehq.pillows.mappings.utils import mapping_from_json
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-APP_INDEX = es_index("hqapps_2019-01-23")
+APP_INDEX = es_index("hqapps_2019-05-17")
```

Comment:
I don't think it's necessary to change the index name - you could just update the mapping since you're just adding a new field. You can update the mapping in a Django migration like this: https://github.com/dimagi/commcare-hq/blob/a13f0e173395a5415df5fe6d06220cfe4d0cabff/corehq/apps/case_search/migrations/0008_auto_20180119_1716.py#L25

Reply:
It looks like that migration was auto-generated, but how? `makemigrations` seems to only deal with sql models. When I try writing the migiration by hand I run into errors about conflicting migrations and attempting to merge them throws a "Could not find common ancestor". The migration looks like this (plus imports):

```
def update_es_mapping(*args, **kwargs):
    def _update_es_mapping():
        return call_command('update_es_mapping', 'app', noinput=True)
    return _update_es_mapping

class Migration(migrations.Migration):
    operations = [
        migrations.RunPython(update_es_mapping)
    ]
```

Reply:
You need a `dependencies` section in the `Migration` class. This can be the name of the previous migration in the same app. 

https://github.com/dimagi/commcare-hq/blob/a13f0e173395a5415df5fe6d06220cfe4d0cabff/corehq/apps/case_search/migrations/0008_auto_20180119_1716.py#L20-L22

Reply:
thanks - e06d064d7df